sparse_strips/vello_hybrid/src/perf_measurement.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright 2025 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Performance measurement functionality for GPU operations.
+
+use std::sync::Mutex;
+use wgpu::{Buffer, BufferUsages, CommandEncoder, Device, QuerySet, QueryType, Queue};
+
+/// Handles performance measurement using GPU timestamp queries
```

Comment:
:eyes:

Was wgpu-profiler evaluated for this use?

Reply:
Nope, but it definitely should be! I’ve removed that custom profiler and will evaluate/add `wgpu-profiler` in a separate PR. Thanks for pointing it out!

Removed in 83148ee